### DISCUSSION TITLE
Disable restart files

### DISCUSSION THREAD

**Discussion by bocaj2222** [OpenRadioss/OpenRadioss#2318] [Q&A]
Hello,

Is there a way to prevent the creation of restart files when running mpi? I've tried used /RFILE/OFF in the engine file without luck. Also, adding -norst to the starter exe causes the engine exe to fail.

Perhaps rst files are required? If so, is there a method to have them automatically removed? If not, I can manage manually.

Thanks,

Jacob

**Comment by MarianBulla** [comment]
Hello bocaj2222,
and thanks for your interest on OpenRadioss.
Yes, the restart files are needed by the engine.
The first restart files are created by the starter. They contain your model, prepared in a database format, which the engine can directly use.
Then the engine is writing the restart fiels, in a frequency, you can define in the engine-control file (_0001.rad) or in the header part of your mode, if you use the "1-file-fromat" for your model.
This can also be very useful, if you want to stop a run and restart it and continue on another day.
Or if your model is not at its final stage after the end-time defined in the 1st engine file. So you can create a 2nd engine file and extend the simulation time.
Anyway, after the run is finished, you can remove all the restart files. They are not needed for postprocessing of the results.

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by bocaj2222** [reply]
Hi Marian,

Understood, thanks for your quick response!
